src/api/api.ts

```diff
@@ -152,7 +152,7 @@ function getManifestValidationErrors(
         form
     )
         .then(res => _extractItem(res).errors)
-        .catch(error => [error.toString()]);
+        .catch(error => error.errors || [error.toString()]);
```

Comment:
so it will try to parse response and get list of errors or fall back to previous behavior.